### ISSUE TITLE
Language auto detect

### ISSUE THREAD

**Issue by diego2k** [Microsoft/moodle-auth_oidc#22]
If a new user is created with the plugin his language is always the default language of moodle. After sign up the browser language would be a better choice. 
thanks

**Comment by samalos** [comment]
Yes, this is a problem for multi-language sites. Any workaround for this?
